camel/responses/agent_responses.py

```diff
@@ -44,3 +44,26 @@ def msg(self):
                 "for a single message in msgs."
             )
         return self.msgs[0]
+
+    @property
+    def logits(self):
+        """Return the logits associated with the first message in msgs.
+
+        This property retrieves the `meta_dict` of the first message in `msgs`,
+        which contains information such as token log probabilities and top log
+        probabilities that were computed during message generation.
+
+        Returns:
+            dict: A dictionary containing the logits or log probability
+            information of the tokens.
+
+        Raises:
+            RuntimeError: If the list of messages (`msgs`) does not contain
+            exactly one message.
+        """
+        if len(self.msgs) != 1:
+            raise RuntimeError(
+                "Property msg is only available "
+                "for a single message in msgs."
+            )
+        return self.msgs[0].meta_dict
```

Comment:
I think this part is not necessary as adding this property introduced some redundant code, we can fetch this information from `response.msgs[0].meta_dict`

Reply:
Thanks for the suggestion, I see your point about redundancy. You're right that we can directly access the meta_dict from response.msgs[0] instead of defining a separate property.